### DISCUSSION TITLE
Mantine Dropzone Compatibility.

### DISCUSSION THREAD

**Discussion by Minhazul-Islam18** [mantinedev/mantine#6948] [Q&A] [answered]
I'm using Mantine V6, and trying to install Dropzone.
But I'm not sure which version is compatible with v6!

Help me find out, Please! I really appreciate any help you can provide.

**Comment by rtivital** [comment, marked as answer]
Your `@mantine/dropzone` version should be the same as `@mantine/core`  version.

**Reply by Minhazul-Islam18** [reply]
Thank you, God bless you.
